content/posts/2020-07-15-tweeting-from-pull-requests/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Tweeting from Pull Requests
+author: Gustavo Barbosa
+twitter: gustavocsb
+layout: post
+lang: en
+path: /blog/tweeting-from-pull-requests
+date: '2020-07-18'
+comments: true
+---
+
+Together with [launching](https://engineering.loadsmart.com/blog/hello-world/) our blog, we have recently created (or better: _reactivated_) our Engineering team’s Twitter account. We are planning to use it as the default broadcaster for the work we do at Loadsmart. And although we may have different channels (as this blog itself is one of them) and possibly different forms of content, Twitter is great at notifying what’s new, regardless of form or platform.
+
+However, one of the biggest concerns I had when we started the idea of sharing our work was to not let our account be managed by a single person or a small group in the company. Having specific people responsible for creating those tweets would likely create bottlenecks that could lead to lack of engagement in generating new content. And to make this initiative successful, we need to make sure content creation comes from everyone. It should be accessible to anyone willing to contribute to it. I believe this is what creates a collective sense of ownership.
+
+## The problem
+
+Twitter is a [microblogging](https://en.wikipedia.org/wiki/Microblogging) platform so I thought: "Why not having a similar experience from regular blogs where multiple authors write multiple articles?". Well, that would be nice. But unfortunately, different from regular blogs, we can only post content from a single account.
+
+The easiest solution would be sharing the credentials, some may think. That would work but that only wouldn’t be enough. Collective ownership is also about making sure decisions are made and agreed upon. We would need at least people reviewing each other’s content.
```

Comment:
1) Take out only and add comma after work and before but: that would work, but that wouldn't be enough.
2) We would at least need people reviewing their peers.